drivers/power/supply/regulator_gpio.c

```diff
@@ -65,7 +65,9 @@ static const struct regulator_ops_s g_regulator_gpio_ops =
   NULL,                         /* get_voltage_sel */
   regulator_gpio_enable,        /* enable */
   regulator_gpio_is_enabled,    /* is_enabled */
-  regulator_gpio_disable        /* disable */
+  regulator_gpio_disable,       /* disable */
+  NULL,                         /* enable pulldown  if reg. disabled */
```

Comment:
revert don't need change

Reply:
It's not a revert though unless I'm missing something? The original file didn't have the last 2 NULLs and the struct's last line was without the comma. As I added the two extra NULLs as this driver doesn't have the pulldown methods I added the comma.

Or is it something else?

Reply:
compiler will auto initialize the last two unassigned fields to zero (NULL) automatically, so we don't need change anything here.

Reply:
Isn't it better to include them for completeness now? It's not wrong as such and means this driver matches the header file? Or is there some knock-on effect that I don't know of?

Happy either way.

Reply:
Since there are patches to remove the unnecessary initialization: https://github.com/apache/nuttx/pull/8016

Reply:
OK. I got caught out today by extra fields added to a struct in code I'd written a year ago (but as yet haven't submitted as a PR). New CODE fields had been between existing ones...so I'm not 100% convinced having a mismatch between the definitions and declarations is actually that good an idea. Is there an actual downside to those NULLs?

However I will revert the change.

Reply:
done